msamp/operators/dist_op/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.10)
+
+project(msampdist LANGUAGES CXX CUDA)
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/)
+
+find_package(CUDA REQUIRED)
+find_package(NCCL REQUIRED)
+
+add_library(msampdist SHARED dist.cpp)
+
+target_include_directories(msampdist PUBLIC ${CUDA_INCLUDE_DIRS} ${NCCL_INCLUDE_DIRS})
+target_link_libraries(msampdist PUBLIC ${CUDA_LIBRARIES} ${CUDA_LIBRARIES})
+
+install(TARGETS msampdist)
```

Comment:
`install(TARGETS msamp_dist)` may be better.

reference: libopencv_dn.so, libsvn_ra_svn-1.so, libmpi_cxx.so

Reply:
Good suggestion. Fixed.